TYPO3.Flow/Classes/TYPO3/Flow/Utility/Algorithms.php

```diff
@@ -10,6 +10,7 @@
 
 require_once(FLOW_PATH_FLOW . 'Resources/PHP/iSecurity/Security_Randomizer.php');
```

Comment:
Nope if you watch the full file you notice it is used in many more places (for actual random data). To remove that there is a separate pull request: #5
The thing is that the secure random number generator is WAY slower than the one we use currently (but also more secure and up to the current knowledge) so generating UUIDs with it is not feasible (functional tests take WAY longer then) so I figured it would make sense to use something else for the UUID generation and found this library. As it supports all types of UUIDs we can fullfil the todo for other types with it as well but in a first step I just wanted to replace it so the secure randomizer change can happen as well.